### ISSUE TITLE
UniformTFQuantization inference speedup not reproducible on ARM v8

### ISSUE THREAD

**Issue by dhingratul** [Tencent/PocketFlow#122]
**Describe the bug**
1. Train compressed model with UniformTFQuantization for 0.01 epochs
2. Do a forward pass on the transformed model and the original model using .pb
3. The original model runs at 20ms on a ARMv8, whereas the transformed model runs at 28ms
**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by jiaxiang-wu** [comment]
@dhingratul 
1. Please provide the execution command for training with `UniformTFQuantLearner`.
2. Why are you using *.pb models for inference on ARMv8? You should use *.tflite models instead, which uses the TensorFlow Lite inference library with optimized performance on mobile devices.
3. When measuring the time consumption, it is often more accurate to run multiple times and compute the average elapsed time. Also, some warm-up runs may be needed to stabilize the measured time.

**Comment by dhingratul** [comment]
1. /scripts/run_local.sh nets/resnet_at_ilsvrc12_run.py --learner uniform-tf  --nb_epochs_rat 0.01
2. I need to benchmark both .pb and .tflite
3. The output is averaged over 1000 runs, ignoring first 5 inferences

**Comment by jiaxiang-wu** [comment]
Do you have results for *.tflite models on ARMv8?

**Comment by dhingratul** [comment]
No, not yet, i haven't used .tflite before, hence trying to figure out how to run that instead of a .pb

**Comment by jiaxiang-wu** [comment]
Based on my understanding (please correct me if anything wrong), *.pb models only simulate quantized operations and only *.tflite models use actual fixed-point operations for inference. Therefore, for quantized models, the speed-up is only observed on *.tflite models. This is different from channel pruned models, where both *.pb and *.models use the same floating-point operations with reduced number of channels for speed-up.

**Comment by dhingratul** [comment]
@jiaxiang-wu The speedups are not observed even with a .tflite file

**Comment by dhingratul** [comment]
@jiaxiang-wu Please see https://github.com/Tencent/PocketFlow/issues/123
